core/iwasm/aot/aot_runtime.c

```diff
@@ -2024,6 +2101,7 @@ invoke_native_with_hw_bound_check(WASMExecEnv *exec_env, void *func_ptr,
             void (*invoke_native)(void *func_ptr, void *exec_env, uint32 *argv,
                                   uint32 *argv_ret) =
                 func_type->quick_aot_entry;
+            exec_env->attachment = attachment;
```

Comment:
Have you found issue here? If added here, had better add `exec_env->attachment = NULL` after `invoke_native`, like wasm_runtime_invoke_native and wasm_runtime_invoke_native_raw.